gc/gc-base/src/main/java/org/projectnessie/gc/base/DistributedIdentifyContents.java

```diff
@@ -87,18 +92,32 @@ private static int getPartitionsCount(GCParams gcParams, List<Reference> referen
   }
 
   private static Map<String, ContentBloomFilter> mergeLiveContentResults(
-      List<Map<String, ContentBloomFilter>> bloomFilterMaps) {
+      List<Map<String, ContentBloomFilter>> bloomFilterMaps, double bloomFilterFpp) {
+    Set<String> mergedContentIds = new HashSet<>();
     Map<String, ContentBloomFilter> output = new HashMap<>();
     bloomFilterMaps.forEach(
         map ->
             map.forEach(
                 (k, v) -> {
                   if (output.containsKey(k)) {
                     output.get(k).merge(v);
+                    mergedContentIds.add(k);
                   } else {
                     output.put(k, v);
                   }
                 }));
+    // Since we merged bloom filters log in case their quality deteriorated
+    mergedContentIds.stream()
+        .forEach(
+            contentId -> {
+              double fpp = output.get(contentId).getExpectedFpp();
+              if (fpp > bloomFilterFpp) {
+                LOGGER.info(
+                    "Fpp of ContentBloomFilter for '{}': {}",
```

Comment:
Could we expose this as a metric too?

Reply:
maybe @ajantha-bhat has it planned already for the 2nd/3rd GC impl phase?

Reply:
no plans for metrics as of now. But always good to have. 
GC results we are storing in table, so we can run SQL get some metrics out of it. But nothing concrete as of now.